src/2_storage.js

```diff
@@ -61,7 +61,17 @@ var retrieveValue = function(value) {
 };
 
 var webStorage = function(perm) {
-	var storageMethod = perm ? localStorage : sessionStorage;
+	var storageMethod;
+	try {
+		storageMethod = perm ? localStorage : sessionStorage;
```

Comment:
so is the issue that referencing localStorage or sessionStorage is returning undefined?